src/app.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react'
 import logo from './logo.svg'
-import './App.css'
```

Comment:
I'm updated this to have it be consistent, let me know if that's not what you wanted 😄